src/main/webapp/add_event_page/add_event.html

```diff
@@ -7,40 +7,43 @@
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="initMap()">
+  <body>
     <div id="header">
       <h1>Add New Event</h1>
     </div>
 
     <div id="form">
-        <form action="/new_event.php">
+        <form>
             <label for="name">Event Name:</label>
             <input type="text" id="name" name="name"><br>
```

Comment:
I think we should require the user to fill out the fields. You can put `require` inside the input tag.